### PR TITLE
HCX-1 small improvements

### PR DESCRIPTION
**Pull request by GideonEXE**

### CODE REVIEW

File: src/org/camunda/latera/bss/console.groovy [hunk 1]
```diff
@@ -56,15 +56,19 @@ class Console {
     return [proc: proc, stdout: stdout, stderr: stderr]
   }
 
+  private Boolean isShortCommand(String key) {
```
Comment (dolfinus) [inline src/org/camunda/latera/bss/console.groovy]: Please use CharSequence instead on String in functions arg types.

File: src/org/camunda/latera/bss/console.groovy [hunk 2]
```diff
@@ -56,15 +56,19 @@ class Console {
...
+    return key.size() == 1
+  }
+
   Map runCommand(Map args, CharSequence command) {
     List arguments = []
-    args.each { k, v ->
-      if (k.size() == 1) {
-        arguments += ["-${k} ${v}"]
+    args.each { key, value ->
+      if (isShortCommand(k)) {
```
Comment (TimAle) [inline src/org/camunda/latera/bss/console.groovy]: `k` is `key` now I suppose.

File: src/org/camunda/latera/bss/logging.groovy [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package org.camunda.latera.bss.logging
 
+import static org.camunda.latera.bss.utils.Constants.*
```
Comment (TimAle) [inline src/org/camunda/latera/bss/logging.groovy]: I think logical order of imports is put this line near lines 5-6.

File: src/org/camunda/latera/bss/utils/Constants.groovy [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+class Constants {
+  // Log level
+  static final DEBUG    = 0
+  static final INFO     = 1
+  static final WARNING  = 2
+  static final ERROR    = 3
+  static final CRITICAL = 4
+  static final DEFAULT  = 5
+
```
Comment (TimAle) [inline src/org/camunda/latera/bss/utils/Constants.groovy]: Extra line?

**Comment by GideonEXE** [comment]
UPD

**Comment by dolfinus** [comment]
Fixed

**Pull request merged**